resources/assets/js/components/JobBuilderWorkEnv/WorkEnvForm.tsx

```diff
@@ -882,7 +893,7 @@ const WorkEnvForm = ({
           />
         </h4>
         <p data-c-margin="bottom(normal)">
-          <FormattedMessage {...formMessages.ourWorkEnvDesc} />
+          {intl.formatMessage(formMessages.ourWorkEnvDesc)}
```

Comment:
Just curious why the swap from `FormattedMessage` to `intl.formatMessage`

Reply:
I was getting an eslint error using the spread syntax.

Reply:
Ah okay, I thought I had added an ignore for that globally but it must be in my draft PR

Reply:
Is Formatted Message preferable?

Reply:
It doesn't really matter, I was just wondering since they both do the same thing